.github/actions/optimized_build/action.yaml

```diff
@@ -29,19 +29,14 @@ runs:
     - name: "Check for WASM binaries larger than allowed limit"
       shell: "sh"
       run: |
-        # Temporarily disable check
-        exit 0
+        large_files="`find "artifacts/" -size +3M | sed 's/ /\n/g'`"
 
-        large_files=`find "artifacts/" -size +3M`
-
-        large_files_count=`wc -l < EOF
-          ${large_files}
-          EOF`
+        large_files_count="`echo -n "${large_files}" | wc -l`"
```

Comment:
have you tested with `find "artifacts/" -size +3M | wc -w` ?

Reply:
I haven't, yet it's a bit prettier when they are on a new line each in the step report. It's a bit more clear to read.

The point of separating it was not only to not lose this information as that happened when we encountered the problem with the file size limits.

TL;DR: This is intentionally split into two evals.